docs/UserGuide.md

```diff
@@ -3,271 +3,432 @@ layout: page
 title: User Guide
 ---
 
-ClientBook is a desktop app for managing client contacts, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ClientBook can get your client management tasks done faster than traditional GUI apps.
+# ClientBook
 
+ClientBook is an application for managing client contacts, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+If you are an insurance agent who can type fast, ClientBook can help _you_ accomplish your client management tasks done faster than traditional GUI apps.
```

Comment:
I think there is a typo here. Should be "...client management tasks **faster** than traditional...".

Reply:
Okay, thank you!